tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -17,8 +17,8 @@ RUN /tmp/scripts/install_openvino.sh -o ${OPENVINO_VERSION} && \
 
 WORKDIR /root
 
-ENV INTEL_OPENVINO_DIR /data/dldt/openvino_${OPENVINO_VERSION}
-ENV LD_LIBRARY_PATH $INTEL_OPENVINO_DIR/deployment_tools/inference_engine/lib/intel64:$INTEL_OPENVINO_DIR/deployment_tools/:$INTEL_OPENVINO_DIR/deployment_tools/ngraph/lib:$INTEL_OPENVINO_DIR/deployment_tools/inference_engine/external/tbb/lib:/usr/local/openblas/lib:$LD_LIBRARY_PATH
+ENV INTEL_OPENVINO_DIR /data/openvino/openvino_${OPENVINO_VERSION}.287
```

Comment:
>287 [](start = 67, length = 3)

should we make the minor version as a parameter also? so we don't need to always change this file.

Reply:
yes, seems like the version should include the minor version.